rules/windows/builtin/win_remote_service.yml

```diff
@@ -0,0 +1,33 @@
+title: Remote Service Creation
+id: 4a3a2b96-d7fc-4cb9-80e4-4a545fe95f46
+status: experimental
+description: Detects remote execution via service creation on the destination host
+author: Jai Minton
+date: 2020/10/05
+references:
+    - https://drive.google.com/file/d/1lKya3_mLnR3UQuCoiYruO3qgu052_iS_/view
+tags:
+    - attack.lateral_movement
+    - attack.persistence
+    - attack.execution
+    - attack.t1543.003 
+logsource:
+    product: windows
+    service: security, system
```

Comment:
Hello @JPMinty !

Thank you for your contribution!

There is an issue with this line: `service: security, system`.
This construction gonna fail on export with some export configurations because the `service` field supposed to have only one service. 
Rules with multiple logsources are constructed with multiple YAML documents in one file and a special operator (`action `).
[Here](https://github.com/Neo23x0/sigma/blob/master/rules/windows/builtin/win_meterpreter_or_cobaltstrike_getsystem_service_installation.yml) is an example of such rule. I've found the closest one with the same `service`'s in use (: 

Could you please refactor your rule using this example?

Reply:
[here](https://github.com/Neo23x0/sigma/wiki/Specification#rule-collections) is also some useful materials from the wiki

Reply:
> 
> 
> Hello @JPMinty !
> 
> Thank you for your contribution!
> 
> There is an issue with this line: `service: security, system`.
> This construction gonna fail on export with some export configurations because the `service` field supposed to have only one service.
> Rules with multiple logsources are constructed with multiple YAML documents in one file and a special operator (`action `).
> [Here](https://github.com/Neo23x0/sigma/blob/master/rules/windows/builtin/win_meterpreter_or_cobaltstrike_getsystem_service_installation.yml) is an example of such rule. I've found the closest one with the same `service`'s in use (:
> 
> Could you please refactor your rule using this example?

Thank you for letting me know @yugoslavskiy .
I've refactored this based on what you've sent through.
Should be meeting the requirements now. Thank you for the example and material wiki also.

Sigmac Testing:

```
$ ./sigmac -t splunk -c ./config/splunk-windows.yml ../rules/windows/builtin/win_remote_service.yml
(((EventCode="4624" Logon_Type="3") NOT ((Source_Network_Address="::1" OR Source_Network_Address="127.0.0.1"))) OR EventCode="4697")
(((EventCode="4624" Logon_Type="3") NOT ((Source_Network_Address="::1" OR Source_Network_Address="127.0.0.1"))) OR EventCode="7045")
```

Reply:
Hello @JPMinty! 

Thank you for modifying the rule!
Could you also remove the "`, system`" from the "`service: security, system`"? 
That would fix the original issue.

Reply:
Woops, that entire section was meant to be removed as it's now covered at the bottom, my mistake.